client/index.jsx

```diff
@@ -20,13 +20,15 @@ class GameDetails extends React.Component {
     this.state = {
       images: [],
       details: '',
+      index: 0,
```

Comment:
I dont see where this new state is being used anywhere, but maybe I missed it. Otherwise everything looks good :)